change/@fluentui-react-breadcrumb-preview-994c2089-e9d3-4795-9e33-88937cf99ea3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29158&commitId=050282369ee61ee8ce674458bcc26af7a7516a2e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Breadcrumb Converged</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Breadcrumb Converged.divider type.focused.chromium.png|538| Changed|
|Breadcrumb Converged.divider type.pressed.chromium.png|538| Changed|
|Breadcrumb Converged.size.default.chromium.png|546| Changed|
|Breadcrumb Converged.divider type.hover.chromium.png|538| Changed|
|Breadcrumb Converged.size.pressed.chromium.png|546| Changed|
|Breadcrumb Converged.size.hover.chromium.png|546| Changed|
|Breadcrumb Converged.divider type.default.chromium.png|538| Changed|
|Breadcrumb Converged.size.focused.chromium.png|546| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>